### PR TITLE
Add JPMS module name for each spec project to naming doc

### PR DESCRIPTION
**Pull request by starksm64**
This is based on the discussion of the 2021-11-03 meeting

Signed-off-by: starksm64 <starksm64@gmail.com>

### CODE REVIEW

**Comment by markt-asf** [comment]
+1 for el, jsp, servlet and websocket

File: names.adoc [hunk 1]
```diff
@@ -2,183 +2,235 @@
...
+|security
 
 |{projectBaseURL}/ee4j.faces[ee4j.faces]
 |Jakarta Server Faces
 |{gitBaseURL}/mojarra[Jakarta Server Faces]
 |faces
+|N/A
 
 .2+|{projectBaseURL}/ee4j.jsp[ee4j.jsp]
 .2+|Jakarta Server Pagers
```
Comment (tjwatson) [inline names.adoc:156]: Pagers typo

**Review by tjwatson — APPROVED** [review]

File: names.adoc [hunk 2]
```diff
@@ -2,183 +2,235 @@
...
 |soap
+|xml.soap
 
 |{gitBaseURL}/jax-ws-api[Jakarta XML Web Services]
 |xml-ws
+|xml.ws
 
 |{gitBaseURL}/jws-api[Jakarta Web Services Metadata]
 |ws-metadata
+|N/A
```
Comment (bstansberry) [inline names.adoc:233]: @starksm64 Wouldn't this be 'jws'?

https://jakarta.ee/specifications/web-services-metadata/3.0/apidocs/jakarta.jws/module-summary.html

Comment (bstansberry) [reply names.adoc:233]: It should be; jakarta.jws:jakarta-jws-api:3.0 has a module-info declaring 'jakarta.jws'.

Comment (starksm64) [reply names.adoc:233]: I thought for EE10 metadata was being folded into Jakarta XML Web Services 4.0, at leas that is what is in the https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan, so I put N/A. I'll double check with @lukasj

Comment (lukasj) [reply names.adoc:233]: Jakarta EE 9* uses module name `jakarta.jws`
for EE `10+`, the API is part of xml ws spec, so the module name is `jakarta.xml.ws`

**Pull request merged**